2024-06-24/minutes.md

```diff
@@ -1,3 +1,196 @@
+
+[Chris] I was just going to ask is that just related to kind of closing style issues or is it a wider variety of things?
+
+[Ruth] We will do some triaging and we will look at stale things. The thing is that that has been more of a focus on previous GitHub weeks and these days we actually have quite a lot less than we probably think we do. I think with Pranshu starting and joining last month, it's also really good opportunity just to look at the processes and the way that we do trash things, what our projects look like. Teams and access rights, all those processes kind of need just to sort of be checked and reviewed and make sure that they work for everybody.
+
+[Onkar] Is it like inviting students to work on small tasks?
+
+[Ruth] Not this one. I think that is a different kind of week, which I do hope we will be able to run at some point, but I think this one will just be the core like. The people in the room who are writers here, Pranshu and we can Just start a plan for the next 6 months and then we'll do it all over again at Christmas and then all over again this time next year and just keep iterating and improving.
+
+[Chris] That's great. It sounds super positive because I've noticed that OWD folks have been doing quite a bit of issue triage and stuff recently. So that's great and then Some of our volunteers have as well like Josh Chen has done absolutely loads of stuff recently closing a load of stale issues. I'm also wondering whether it would be worth spending a bit of your time kind of figuring out a way to highlight issues which would be good kind of You know. Like good 1st issue type things but not necessarily that but you know just sort of this is something that can be worked on without a massive mind field hassle, go crazy and run away screaming.
+
+[Ruth] Yeah.
+
+[Chris] I say this not purely just for kind of you know, regular. Flow of. Volunteer contributions but also it's worth having a bunch of those in hand for when you start to get efforts like Hacktoberfest and stuff piling onto your site.
+
+[Ruth] Yeah.
+
+[Chris] Because if they can't find anything obvious to work on, they tend to start making all sorts of ludicrous unnecessarily small changes just so they can get their commit numbers up, which drives everyone crazy. So it's good to preempt things like that as well. I speak from experience having suffered through this for many years when I was working on MDN at Mozilla.
+
+[Ruth] Yeah, so it's our previous community manager. We haven't had one for yet, but he did set up a project specifically for contributors to come in. So I think it would be good idea to revisit that and see whether we can improve it. And Yeah, look at look at the processes for. When people come in, there was a comment recently that our contribution docs take you to get her and then take you back to the contribution docks and that flow is a little bit disjointed. So things like that I think we can look at. Say yes.
```

Comment:
This can probably be combined with one of Ruth's previous comments
```suggestion
[Ruth] Our previous community manager set up a project specifically for contributors. It would be a good idea to revisit that and see whether we can improve it. We also want to improve the workflow between contribution docs on MDN and what we have on GitHub because the flow is a little bit disjointed at the moment.
```